docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/wearable_circularindexscrollbar.htm

```diff
@@ -32,6 +32,16 @@ <h1>Circular Index Scroll Bar</h1>
 	</tbody>
 </table>
 
+<img src="../../images/Circularindexscrollbar.png" alt="photo of circleindex element in Tizen">
```

Comment:
I think we don't need change the alt.
Because the component is optimized for circular UI. The component may not be shown propertly in rectangular UI. 
So I think "circleindex" is proper.